vignettes/creating_linters.Rmd

```diff
@@ -274,6 +274,21 @@ and so they've been tested and demonstrated their utility already.
    extract the string exactly as R will see it. This is especially important to make
    your logic robust to R-4-style raw strings like `R"-(hello)-"`, which is otherwise
    difficult to express, for example as an XPath.
+ * `xml_find_function_calls()`: Whenever your linter needs to query R function calls,
```

Comment:
Maybe note this in the Wiki too?

https://github.com/r-lib/lintr/wiki/Writing-robust,-performant-linters

Reply:
Done.